.github/workflows/build-packages.yaml

```diff
@@ -93,11 +93,17 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Install yq (for Fedora spec generation)
+        run: sudo apt-get install -y yq
+
       - name: Download artifacts
         uses: actions/download-artifact@v3
         with:
           path: downloaded-artifacts/
 
+      - name: Generate spec files for COPR submission
+        run: ./scripts/generate_spec.sh
+
```

Comment:
Previously the `create-release` step was only actually used for updating the test release with new packages, and all the stable releases were created manually. I've changed this in https://github.com/ilya-zlobintsev/LACT/commit/23abed8ad6c718b5158dc637a83f5faec80a2607 so now there are two steps, one for all commits and one for stable tags. After this change the spec generation should go into the tag creation step, since I assume for now the COPR package will be just for stable versions.